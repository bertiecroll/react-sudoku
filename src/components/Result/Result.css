.Result {
  margin: 1em;
  padding: 1em;
  display: flex;
  flex-direction: column;
  align-items: center;
  color: #999;
}

.Result h2 {
  margin-bottom: .1em;
}

.Result p {
  margin-top: .1em;
}

.Result a {
  width: 12em;
  margin: .5em;
  padding: .75em;
  text-align: center;
  border-radius: .5em;
  text-decoration: none;
  background-color: #67DAF9;
  box-shadow: 1px 1px 1px 0px #000;
  color: #fff;
}
