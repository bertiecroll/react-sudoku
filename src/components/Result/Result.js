import React from 'react'
import { Link } from 'react-router-dom'
import './Result.css'

export default function Result (props) {
  return (
    <div className="Result">
      <h2>Well Done!</h2>
      <p>Puzzle completed</p>
      <Link to="/">Play Again</Link>
    </div>
  )
}
