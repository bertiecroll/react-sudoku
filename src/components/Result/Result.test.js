import React from 'react';
import { shallow } from 'enzyme'
import Result from './Result';

describe('Result', () => {
  it('renders component content', () => {
    const wrapper = shallow(<Result />)
    expect(wrapper).toMatchSnapshot()
  })
})
