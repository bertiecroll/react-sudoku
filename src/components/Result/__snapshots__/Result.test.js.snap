// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result renders component content 1`] = `
<div
  className="Result"
>
  <h2>
    Well Done!
  </h2>
  <p>
    Puzzle completed
  </p>
  <Link
    replace={false}
    to="/"
  >
    Play Again
  </Link>
</div>
`;
